File: Resource/51900286-51900712-51900718-51900699/source demo/Window Form/51900286-51900712-51900718-51900699/RestaurentDTO/NhaCungCap.cs
using System;
using System.Data;

namespace RestaurentDTO
{
    public class NhaCungCap
    {
        private string _maNCC;
        private string _tenNCC;
        private string _moTaNCC;
        private string _diaChi;
        private string _soDienThoai;

        public NhaCungCap(DataRow dataRow)
        {
            _maNCC = dataRow["MaNhaCungCap"].ToString();
            _tenNCC = dataRow["TenNhaCungCap"].ToString();
            _moTaNCC = dataRow["MoTaNhaCungCap"].ToString();
            _diaChi = dataRow["DiaChi"].ToString();
            _soDienThoai = dataRow["SoDienThoai"].ToString();
        }

        public string MaNCC { get => _maNCC; set => _maNCC = value; }
        public string TenNCC { get => _tenNCC; set => _tenNCC = value; }
        public string MoTaNCC { get => _moTaNCC; set => _moTaNCC = value; }
        public string DiaChi { get => _diaChi; set => _diaChi = value; }
        public string SoDienThoai { get => _soDienThoai; set => _soDienThoai = value; }
    }
}

File: Dev/Models/Models/NhaCungCap.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="nhacungcap")
public class NhaCungCap {
    @Id
    @GeneratedValue
    private Long MaNhaCungCap;
    private String MoTaNhaCungCap;
    private String TenNhaCungCap;
    private String DiaChi;
    private String SoDienThoai;
}